macros/src/lib.rs

```diff
@@ -121,26 +121,3 @@ pub fn maybe_await(expr: TokenStream) -> TokenStream {
 
     quoted.into()
 }
-
```

Comment:
Since you've changed the `bdk-macros` you should also update the `Cargo.toml` accordingly and use the local version. Otherwise when it's time to release this changes we will probably forget about them